sniffles/petabi_rule_writer/petabi_rule_writer.py

```diff
@@ -150,14 +181,37 @@ def check_pcre_compile(re):
     return True
 
 
+# Formats background traffic rule to a petabi rule format.
+# There can only be 1 background traffic rule in a whole rule file.
+# Output: foramtted background traffic rule or None
+def formatBackgroundTrafficRule(background_traffic=None):
```

Comment:
This default value of `None` doesn't do any good. The following statement will always be evaluated as `None` and waste CPU time.
```python
result = formatBackgroundTrafficRule()
```
Better to make such a call cause an error, so that the caller knows it must supply this function with an actual argument.